### ISSUE TITLE
Game does not update you name to the current steam displayname

### ISSUE THREAD

**Issue by Sennel** [Bouncyrock/Temp-Public-TS-Issue-Tracker#4]
When starting the game via steam, it sets your current display name to be your name *FOREVER*

**Comment by cbaggers** [comment]
Yes, that is by design currently so people can have their own alias within talespire.

If we support having steam change this then we will have slight funtionality clashes in future as peopel who log without using steam will be able to change their name whenever from inside TS, but steam users wont be able to.

I'm open for a discussion on this though. Maybe that limitation is ok for steam users?

**Comment by Sennel** [comment]
As long as my first login with steam doesn't set anything in stone it seems fine. Although later on you might want to enable name changes via the steam display name.

**Comment by cbaggers** [comment]
Definitely. Thanks @Sennel

**Comment by cbaggers** [comment]
internal ticket https://github.com/Bouncyrock/TaleSpire/issues/267
